src/cast-playback-engine.js

```diff
@@ -198,7 +198,7 @@ class CastPlaybackEngine extends FakeEventTarget {
   _maybeDispatchTimeUpdate(): void {
     if (!this._seeking) {
       this.dispatchEvent(new FakeEvent(EventType.TIME_UPDATE));
-    } else if (this._seekTargetTime && this._currentTime >= this._seekTargetTime) {
+    } else if (this._seekTargetTime && this._currentTime === this._seekTargetTime) {
```

Comment:
a bit risky to have it pin point exact time IMO.
Let's make it more error prune and add seek direction detection.

Reply:
done